pymc3/step_methods/smc.py

```diff
@@ -28,7 +28,7 @@
 from .arraystep import metrop_select
 from ..backends import smc_text as atext
 
-__all__ = ['SMC', 'ATMIP_sample']
+__all__ = ['SMC', 'smc_sample']
```

Comment:
The other dedicated sampling functions are named the other way round: `sample_gp`, `sample_ppc`.